moabb/tests/evaluations.py

```diff
@@ -206,6 +207,7 @@ class Test_WithinSessLearningCurve(unittest.TestCase):
     initialization instead of during running the evaluation
     """
 
+    @pytest.mark.skip(reason="This test is not working")
     def test_correct_results_integrity(self):
```

Comment:
This test is not working at all, and looks like was not executed with the previous configuration

Reply:
I opened an issue to work on this later

Reply:
yes, we could work on that later